compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.td

```diff
@@ -142,6 +142,18 @@ def LLVMCPUTileAndFuse :
   ];
 }
 
+def LLVMCPUVerifyVectorSizeLegality :
+    InterfacePass<"iree-llvmcpu-verify-vector-size-legality", "mlir::FunctionOpInterface"> {
+  let summary =
+      "Signals errors when there are large vectors in the IR. I.e., one of"
+      "vector sizes is greater than ratio * native_vector_size";
+  let options = [
+    Option<"ratio", "ratio", "int64_t", /*default=*/"512",
+           "The ratio used in the computation of max vector size.">
```

Comment:
It's not clear to me what "ratio" means in this context. IIUC, this pass makes sure that for every vector `v` in the input IR:

*  `number of bytes(v)` < `ratio` * `native_vector_size`?

So that would be `maxAllowedNumberOfNativeVectors`? Did I get this correctly?

Reply:
I don't like variable names being so long, but I could not get a better name. `maxAllowedNumberOfNativeVectors` is definitely better than `ratio`, so I followed the suggestion. Thanks!